pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -229,6 +236,9 @@ func (m *Multicluster) initializeCluster(cluster *multicluster.Cluster, kubeCont
 		// Add an instance handler in the kubernetes registry to notify service entry store about pod events
 		kubeRegistry.AppendWorkloadHandler(m.serviceEntryController.WorkloadInstanceHandler)
 	}
+	if m.configController != nil {
```

Comment:
If we need the `isNil` check elsewhere, I'd suppose it's needed here? Or is the case where that check fails due to something we only due in a test?

Reply:
TBH, I don't know... the other one definitely needs it